iis-wf/iis-wf-report/src/test/resources/eu/dnetlib/iis/wf/report/builder/test/input/processing_times_report.json

```diff
@@ -1,4 +1,4 @@
-{"key":"processing.citationMatching.total.duration", "type":"DURATION", "value": "22336000"}
```

Comment:
Let's drop `total` here (instead of renaming to `deduped`) as already justified in one of the comments above.